lib/iex/test/test_helper.exs

```diff
@@ -1,6 +1,10 @@
+assert_timeout = String.to_integer(
+  System.get_env("ELIXIR_ASSERT_TIMEOUT") || "500"
```

Comment:
It can also be written as:

    assert_timeout =
      String.to_integer(System.get_env("ELIXIR_ASSERT_TIMEOUT") || "500")

Which is my preferred.